db/migrate/20150714132601_remove_is_param.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveIsParam < ActiveRecord::Migration
+  def up
+    remove_column :lookup_keys, :is_param
+  end
+
+  def down
+    add_column :lookup_keys, :is_param, :boolean
+  end
+end
```

Comment:
what about user permissions migrations? I would assume if you had permissions on the old name you should also have permission on the new name?